pkg/controller/clusterprovision/clusterprovision_controller.go

```diff
@@ -583,3 +589,53 @@ func (r *ReconcileClusterProvision) deleteInstallJob(provision *hivev1.ClusterPr
 	}
 	return reconcile.Result{}, nil
 }
+
+// getWorkers fetches the number of worker replicas from the installConfig, and reports "unknown" if not available
+func (r *ReconcileClusterProvision) getWorkers(cd hivev1.ClusterDeployment) string {
+	icSecret := &corev1.Secret{}
+	err := r.Get(context.Background(),
+		types.NamespacedName{
+			Namespace: cd.Namespace,
+			Name:      cd.Spec.Provisioning.InstallConfigSecretRef.Name,
+		},
+		icSecret)
+	if err != nil {
+		r.logger.WithError(err).Log(controllerutils.LogLevel(err), "Error loading install config secret")
```

Comment:
I think this should be
```suggestion
		r.logger.WithError(err).Error("Error loading install config secret")
```
I think in other places we use `Log(LogLevel(err), ...)` so we don't emit the log for certain kinds of `err` that might be "expected" etc. But in this case we definitely want to log at error level.